lib/api/api.js

```diff
@@ -1865,6 +1890,6 @@ api.prototype.isReattachable = function(inputAddresses, callback) {
             return callback(null, results);
         }
     })
-}
+};
```

Comment:
Adding semicolons is a small thing, and it's technically unnecessary.  But, it clears out a TON of false positives from my IDE's inspection warnings, making it easier to focus on the lines that really need attention.